### ISSUE TITLE
Faulty heading formatting

### ISSUE THREAD

**Issue by bomhard** [pmoreno-rodriguez/grav-theme-future2021#39]
Hi, great theme.

H1 (#) is rendered smaller than H2 (##). I found that in main.css there is no size given for h1. Adding a style definition for h1 = 1.3em there doesn't change anything. I also changed it in main.css.min without success.

Commenting out the uppercase style for Hx is working.

Tested on actual Firefox and Chrome on Debian 12.

Your theme 'Editorial' behaves as expected as well as 8 other themes I tested.

Best
Nikolaus

**Comment by pmoreno-rodriguez** [comment]
Hello @bomhard 

Thanks for your comments. In principle, the styles used are those of the original HTMlL5Up theme, although some modifications have been incorporated in the latest versions.

I will take your suggestion into account for the next version.

**Comment by bomhard** [comment]
Hello,

I checked it a little deeper. The original main.css has no definition for h1, only #header h1.

Adding 

h1 { font-size: 1.35em;
}

worked. I made a mistake with comments so first id did not. I think fixing that makes sense. Thanks
again for your work.

Best
Nikolaus

PS There is also a problem with that theme and ' Plugin: Editable with ContentTools'. (Edit button
only show when right sidebar is visible. Should I open another issue?

-------- Ursprüngliche Nachricht --------
Von: pmoreno.rodriguez ***@***.***>
Antwort an: pmoreno-rodriguez/grav-theme-future2021
***@***.***>
An: pmoreno-rodriguez/grav-theme-future2021 ***@***.***>
Kopie: bomhard ***@***.***>, Mention ***@***.***>
Betreff: Re: [pmoreno-rodriguez/grav-theme-future2021] Faulty heading formatting (Issue #39)
Datum: 08.07.2023 17:47:47

Hello @bomhard
Thanks for your comments. In principle, the styles used are those of the original HTMlL5Up theme,
although some modifications have been incorporated in the latest versions.
I will take your suggestion into account for the next version.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: <pmoreno-rodriguez/grav-theme-
***@***.***>

**Comment by pmoreno-rodriguez** [comment]
Actually, no theme is 100% adapted to all plugins, but taking into account that each developer creates their plugins to offer a functionality, tested in a particular theme (for example, Quark). It's hard to see why the Future2021 theme doesn't fit well with this plugin.

I myself had to create a template for the owl-carousel plugin to work with Future2021, but I'm afraid I won't be able to do the same for all plugins that don't fit the theme.

In any case, you are welcome to improve the theme, through a Pull request or through suggestions.

**Comment by pmoreno-rodriguez** [comment]
Hi @bomhard 

Try adding this lines to misc.css:

```
/* Support for Editable with ContentTools plugin */
.ct-widget.ct-ignition {
  z-index: 10015;
}
```
The editable buttons will appear now in front of all elements. Now you can decide if the position is what you want, or if you prefer other position, you can play changing some parameters such as right, bottom, left, etc.

**Comment by bomhard** [comment]
Thanks a lot. That is working nicely.

-------- Ursprüngliche Nachricht --------
Von: pmoreno.rodriguez ***@***.***>
Antwort an: pmoreno-rodriguez/grav-theme-future2021
***@***.***>
An: pmoreno-rodriguez/grav-theme-future2021
***@***.***>
Kopie: bomhard ***@***.***>, Mention
***@***.***>
Betreff: Re: [pmoreno-rodriguez/grav-theme-future2021] Faulty heading
formatting (Issue #39)
Datum: 10.07.2023 22:42:48

Hi @bomhard
Try adding this lines to misc.css:
/* Support for Editable with ContentTools plugin */
.ct-widget.ct-ignition {
  z-index: 10015;
}
The editable buttons will appear now in front of all elements. Now you
can decide if the position is what you want, or if you prefer other
position, you can play changing some parameters such as right, bottom,
left, etc.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: <pmoreno-
***@***.***>

**Comment by pmoreno-rodriguez** [comment]
Solved in new version 1.0.6
